integrations/mongodb_atlas/src/haystack_integrations/document_stores/mongodb_atlas/document_store.py

```diff
@@ -168,6 +210,7 @@ def embedding_retrieval(
         :param query_embedding: Embedding of the query
         :param filters: Optional filters.
```

Comment:
Should we document that filters are currently not implemented / supported?

Reply:
The PR for filters will be ready very soon, so we can skip this, I think.

Reply:
ah okay 👍🏻